package.json

```diff
@@ -45,6 +45,7 @@
         "test": "react-scripts test --env=jsdom --silent",
         "test:ci": "cross-env CI=true npm run test",
         "test:coverage": "npm run test -- --coverage",
+        "plato": "./node_modules/.bin/tsc --noEmit false --outDir js-src && ./node_modules/.bin/plato -r -d plato-report js-src",
```

Comment:
You shouldn't need the prefix of `node_modules` on these.  If the package is installed locally within the project node will look up the reference in the local node_modules before looking into any global modules.